src/assets/img/icons/add-to-dm.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M17 8C17 10.7614 14.7614 13 12 13C9.23858 13 7 10.7614 7 8C7 5.23858 9.23858 3 12 3C14.7614 3 17 5.23858 17 8ZM15.5 8C15.5 9.933 13.933 11.5 12 11.5C10.067 11.5 8.5 9.933 8.5 8C8.5 6.067 10.067 4.5 12 4.5C13.933 4.5 15.5 6.067 15.5 8Z" fill="black"/>
+<path d="M6 11.8C4.4536 11.8 3.2 10.5464 3.2 9C3.2 7.4536 4.4536 6.2 6 6.2C6.47563 6.2 6.92355 6.31859 7.31583 6.52782L7.78577 5.41977C7.24825 5.15114 6.64178 5 6 5C3.79086 5 2 6.79086 2 9C2 11.2091 3.79086 13 6 13C7.2416 13 8.35108 12.4343 9.08475 11.5466L8.13554 10.811C7.62194 11.4161 6.85581 11.8 6 11.8Z" fill="black"/>
+<path d="M18.085 11.8C19.6314 11.8 20.885 10.5464 20.885 9C20.885 7.4536 19.6314 6.2 18.085 6.2C17.6093 6.2 17.1614 6.31859 16.7691 6.52782L16.2992 5.41977C16.8367 5.15114 17.4432 5 18.085 5C20.2941 5 22.085 6.79086 22.085 9C22.085 11.2091 20.2941 13 18.085 13C16.8434 13 15.7339 12.4343 15.0002 11.5466L15.9494 10.811C16.463 11.4161 17.2292 11.8 18.085 11.8Z" fill="black"/>
+<path d="M12.1099 17.5007C12.5288 17.506 12.8997 17.2015 12.9345 16.7839C12.9685 16.3762 12.6689 16.0138 12.2599 16.0033C12.1735 16.0011 12.0868 16 11.9998 16C8.95932 16 6.23574 17.357 4.40167 19.4984C4.16181 19.7784 4.19319 20.1933 4.45392 20.454C4.77772 20.7779 5.31374 20.7312 5.61624 20.3874C7.17395 18.6171 9.45645 17.5 11.9998 17.5C12.0366 17.5 12.0732 17.5002 12.1099 17.5007Z" fill="black"/>
+<path d="M15.25 16.75C15.25 16.3358 15.5858 16 16 16H17.25C17.5261 16 17.75 15.7761 17.75 15.5V14.25C17.75 13.8358 18.0858 13.5 18.5 13.5C18.9142 13.5 19.25 13.8358 19.25 14.25V15.5C19.25 15.7761 19.4739 16 19.75 16H21C21.4142 16 21.75 16.3358 21.75 16.75C21.75 17.1642 21.4142 17.5 21 17.5H19.75C19.4739 17.5 19.25 17.7239 19.25 18V19.25C19.25 19.6642 18.9142 20 18.5 20C18.0858 20 17.75 19.6642 17.75 19.25V18C17.75 17.7239 17.5261 17.5 17.25 17.5H16C15.5858 17.5 15.25 17.1642 15.25 16.75Z" fill="black"/>
+</svg>
```

Comment:
Kinda unclear how this icon is needed for the change in the commit

Reply:
> Kinda unclear how this icon is needed for the change in the commit

Divided into different commits!